.github/scripts/auto-label.cjs

```diff
@@ -156,7 +156,7 @@ const getChangedFiles = async (github, identifier) => {
 const parseChangedLang = (patch) => {
   const set = new Set();
   for (const lang of ['cn', 'de', 'ja', 'fr', 'ko']) {
-    const pattern = new RegExp(`^\s*[+].*(?:${lang}|'${lang}'): `);
+    const pattern = new RegExp(String.raw`^\s*[+]\s*(?:${lang}|'${lang}'): `);
```

Comment:
```suggestion
    const pattern = /^[+]\s*(?:${lang}|'${lang}'): /;
```

What about just using `/regex/` as a simplification? Also I think I suggested this, but Id on't think there's any whitespace before the initial `+`.

Reply:
not working

Reply:
Maybe it makes sense to add https://eslint.org/docs/rules/prefer-regex-literals ?

Reply:
Oh right sorry, template interpolation.  Sorry, I should have more coffee before making suggestions.  <_<